CSS/style.css

```diff
@@ -631,3 +608,442 @@ section.contact_me_mobile > hr {
+}
+
+.desk_main_content {
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 1rem;
+}
+
+.desk_main_content h1 {
+  font-size: 40px;
+  font-weight: bold;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  color: #172b4d;
+  text-align: left;
+  margin-top: 1rem;
+}
+
+.desktop-section .desk_main_content p {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
+  height: 50px;
+  color: #344563;
+  text-align: left;
+  margin-top: 2rem;
+  margin-left: 1px;
+}
+
+.desk_main_img1 {
+  display: grid;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  border-radius: 8px;
+}
+
+.desk_main_img2 {
+  display: flex;
+  background-image: url('../Images/deskcard2.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img3 {
+  display: grid;
+  background-image: url('../Images/deskcard3.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  border-radius: 8px;
+}
+
+.desk_main_img4 {
+  display: flex;
+  background-image: url('../Images/deskcard4.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+section.desktop_ams {
+  display: flex;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-top: 5rem;
+  padding-bottom: 10rem;
+  padding-left: 5rem;
+  border-top-right-radius: 150px;
+  gap: 1rem;
+}
+
+section.desktop_ams .about_part1 {
+  width: 40%;
+}
+
+section.desktop_ams .about_part2 {
+  width: 60%;
+}
+
+section.desktop_ams .about_part1 h2 {
+  display: flex;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  color: #091e42;
+  text-align: left;
+  margin-top: 10px;
+}
+
+section.desktop_ams .about_part1 p {
+  display: flex;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  height: 114px;
+  margin-left: 1px;
+  color: #344563;
+  text-align: left;
+  margin-top: 10px;
+}
+
+section.desktop_ams .about_part2 ul {
+  list-style: none;
+}
+
+section.desktop_ams .about_part2 ul li ul {
+  display: flex;
+  list-style: none;
+}
+
+.letconnect_icons ul li:hover {
+  transform: scale(1.1);
+}
+
+.centered4 ul li:hover {
+  transform: scale(1.1);
+}
+
+section.desktop_ams .about_part2 ul li ul li {
+  display: flex;
+  list-style: none;
+  width: 140px;
+  height: 120px;
+}
+
+.contact_me_desktop {
+  display: grid;
+  justify-content: center;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  background-color: #6070ff;
+  border-top-left-radius: 150px;
+  margin-bottom: 2rem;
```

Comment:
- [OPTIONAL] As requested by the previous reviewer please remove the `margin-bottom: 2rem;` from the class `.contact_me_desktop` it's presenting an unnecessary space down below the contact section that needs to be removed.

![image](https://user-images.githubusercontent.com/68030297/229302960-2c5722a0-d801-4a2f-ae06-1a00472f5136.png)

Reply:
@adel-gu - Thank you for your kind approval. I removed the margin bottom value from the contact me page.
Thanks